src/Card/README.md

```diff
@@ -670,3 +671,53 @@ it is meant to be used as a single horizontal row of Cards, not as a grid. See C
   </Card>
 </CardDeck>
 ```
+
+### Card status
```

Comment:
I think the heading level should be up a level for the heading hierarchy on the page? As is, it looks like it's nested under `CardDeck`. Also, it might make sense to put "Card status" section further up on the page, maybe after "Horizontal variant".

Reply:
Placed under `horizontal` variant and made it as `h2` heading